aten/src/ATen/native/cuda/MultiTensorApply.cuh

```diff
@@ -0,0 +1,101 @@
+#include <torch/cuda.h>
+#include <ATen/ATen.h>
```

Comment:
I bet this is causing the Windows error. Stop including so much stuff, just include what you need; if you wildly include every header in PyTorch, this probably makes it more likely you'll trigger an nvcc bug.

Reply:
Done! 
removed cuda.h and torch.h from a different file, build seems to be green now.